src/migration-failure-worker/interface.ts

```diff
@@ -7,4 +7,6 @@ export interface Message {
   migrationFailed?: boolean;
   migrationId: number;
   source?: string;
+  service?: string;
```

Comment:
This file needs to be renames into something like Message.ts and optionally moved into a sub folder of interfaces. The name interface.ts is vague and does not necessarily clarify what the interface is all about.

Reply:
done.